src/iOS/toga_iOS/libs/uikit.py

```diff
@@ -326,6 +326,49 @@ class UIProgressViewStyle(Enum):
 # UISwitch.h
 UISwitch = ObjCClass('UISwitch')
 
+######################################################################
+# UITabBar.h
+
+UITabBar = ObjCClass('UITabBar')
+
+UITabBarItemPositioningAutomatic = 0
+UITabBarItemPositioningFill = 1
+UITabBarItemPositioningCenterd = 2
```

Comment:
Typo: Centerd

Reply:
Will change to `UITabBarItemPositioningCentered = 2`